packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/views/explorer/services/ExplorerDescriptionProvider.java

```diff
@@ -324,10 +325,10 @@ private Object getParentObject(VariableManager variableManager) {
         var optionalEditingContext = variableManager.get(IEditingContext.EDITING_CONTEXT, IEditingContext.class);
         Object result = null;
 
-        if (self instanceof RepresentationMetadata && optionalTreeItemId.isPresent() && optionalEditingContext.isPresent()) {
+        if ((self instanceof IRepresentation || self instanceof RepresentationMetadata) && optionalTreeItemId.isPresent() && optionalEditingContext.isPresent()) {
```

Comment:
_self_ should be `RepresentationMetadata`.
This will need a change in `DefaultObjectSearchService#getRepresentation` to make it returns `RepresentationMetadata` instead of the `IRepresentation`

Reply:
This is not working because this `RepresentationMetadata` is the one from sirius-web-domain and the one put in the variable self by `DefaultEObjectSearchService` is the one from sirius-components-core.
So, I will try implementing a `IObjectSearchServiceDelegate` in sirius-web-application which will retrieve the `RepresentationMetadata` from sirius-web-domain if the given object ID is a representation ID.

Reply:
I will put that fix on hold, because in almost all cases we want the IObjectService#getObject to return a `RepresentationMetadata` but not in the context of the DiagramFilter which display a Form containing diagram elements in a Tree. In that case we want the object service to return the IRepresentation.